addons/l10n_in_ewaybill_stock/models/stock_move.py

```diff
@@ -0,0 +1,57 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+    _description = "Stock Move Ewaybill"
+
+    ewaybill_id = fields.Many2one(
```

Comment:
Should not it be one per picking?

Reply:
Yes it is one per picking but we need it to show the move on `l10n_in_ewaybill` (lines)